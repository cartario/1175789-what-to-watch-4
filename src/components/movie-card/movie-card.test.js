import React from "react";
import renderer from "react-test-renderer";
import MovieCard from "./movie-card";

const film = {
  title: ``,
  src: ``,
};

it(`should render movieCard`, () => {
  const tree = renderer
    .create(
        <MovieCard
          film = {film}
          handleHover = {() => {}}
        />)
    .toJSON();

  expect(tree).toMatchSnapshot();
});
