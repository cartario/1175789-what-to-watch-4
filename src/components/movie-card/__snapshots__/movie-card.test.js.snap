// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MovieCard-test should render movie-card 1`] = `
<article
  className="small-movie-card catalog__movies-card"
  onMouseLeave={[Function]}
  onMouseOver={[Function]}
>
  <div
    className="small-movie-card__image"
  >
    <video />
  </div>
  <h3
    className="small-movie-card__title"
  >
    <a
      className="small-movie-card__link"
      href="/films/1"
      onClick={[Function]}
    >
      
    </a>
  </h3>
</article>
`;
